src/components/Header/Button/Button.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import BtnCSS from './Button.module.css';
+
+const Button = ()=>(
+        <div className={BtnCSS.div_btn}>
```

Comment:
the same here.
Also, if it button - use **button** html element

Reply:
Fixed